### PR TITLE
Don't show right sidebar if "hugeTable: true" is set

### PR DESCRIPTION
**Pull request by dholbach**
Fixes: #178 

Modify import script for this. Docs layout as well, notable diff to `docsy` is:

```diff
--- themes/docsy/layouts/docs/baseof.html	2021-04-28 16:57:40.000000000 +0200
+++ layouts/docs/baseof.html	2021-05-25 17:07:32.433230910 +0200
@@ -13,10 +13,14 @@
           <div class="col-12 col-md-3 col-xl-2 td-sidebar d-print-none">
             {{ partial "sidebar.html" . }}
           </div>
+          {{ if not .Params.hugeTable }}
           <div class="d-none d-xl-block col-xl-2 td-toc d-print-none">
             {{ partial "toc.html" . }}
           </div>
           <main class="col-12 col-md-9 col-xl-8 pl-md-5" role="main">
+          {{ else }}
+          <main class="col-12 col-md-12 col-xl-10 pl-md-5 " role="main">
+          {{ end }}
             {{ partial "version-banner.html" . }}
             {{ if not .Site.Params.ui.breadcrumb_disable }}{{ partial "breadcrumb.html" . }}{{ end }}
             {{ block "main" . }}{{ end }}
```

### CODE REVIEW

**Comment by dholbach** [comment]
- https://deploy-preview-281--fluxcd.netlify.app/docs/migration/timetable/ vs https://fluxcd.io/docs/migration/timetable/
- https://deploy-preview-281--fluxcd.netlify.app/docs/use-cases/azure/ on a normal page

**Review by stefanprodan — APPROVED** [review]
The migration table looks readable now! Thanks @dholbach 🍻

File: layouts/docs/baseof.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            {{ if not .Site.Params.ui.breadcrumb_disable }}{{ partial "breadcrumb.html" . }}{{ end }}
+            {{ block "main" . }}{{ end }}
+          </main>
+        </div>
+      </div>
+      {{ partial "footer.html" . }}
+    </div>
+    {{ partial "scripts.html" . }}
+  </body>
+</html>
```
Comment (hiddeco) [inline layouts/docs/baseof.html:34]:
```suggestion
</html>

```

Comment (dholbach) [reply layouts/docs/baseof.html:34]: That was straight-copied from `docsy` and I applied the minimal diff, but sure whatever.

**Pull request merged**